bot.py

```
@@ -0,0 +1,610 @@
+        max_height = max(heights)
+
+        new_im = Image.new('RGBA', (total_width, max_height))
+
+        x_offset = 0
+        for im in images:
+          new_im.paste(im, (x_offset,0))
+          x_offset += im.size[0]
+
+        new_im.save('./images/enemies.png')
+
+        await message.channel.send(file = discord.File( "./images/enemies.png"))
```

Ruff (0.14.0)

`message` may be undefined, or defined from star imports

(F405)